backend/oasst_backend/prompt_repository.py

```diff
@@ -888,14 +888,27 @@ def fetch_message_with_max_children(self, message: Message | UUID) -> tuple[Mess
         max_message = max(tree, key=lambda m: m.children_count)
         return max_message, [m for m in tree if m.parent_id == max_message.id]
 
-    def _add_user_emojis_all(self, qry: Query) -> list[Message]:
```

Comment:
`_add_user_emojis_all` is a function that is used in several places as a wrapper around Message queries .. IMO this is a bit hacky to add an `include_user` param which is only needed at one place.